cmd/antrea-agent/agent.go

```diff
@@ -766,7 +766,7 @@ func run(o *Options) error {
 			o.config.ClientConnection, o.config.KubeAPIServerOverride,
 			k8sClient, localPodInformer.Get(),
 			podUpdateChannel,
-			&o.config.SecondaryNetwork, ovsdbConnection)
+			&o.config.SecondaryNetwork, ovsdbConnection, ifaceStore)
```

Comment:
It would be safer to initialize secondary interface store and subscribe to the podUpdateChannel before starting the primary CNI Servers (CNIServer.Run()), so no CNI update would be missed. Could you check if possible to move secondary network init to be before CNIServer.Run()?